appserver/tests/application/src/main/java/org/glassfish/main/test/app/security/jmac/https/HttpsTestAuthModule.java

```diff
@@ -99,7 +99,7 @@ public AuthStatus validateRequest(final MessageInfo messageInfo, final Subject c
 
     @Override
     public AuthStatus secureResponse(final MessageInfo messageInfo, final Subject serviceSubject) throws AuthException {
-        return AuthStatus.SUCCESS;
+        return AuthStatus.SEND_SUCCESS;
```

Comment:
That one doesn't need to be overriden, it's already the default:

default AuthStatus secureResponse(MessageInfo messageInfo, Subject serviceSubject) throws AuthException {
        return AuthStatus.SEND_SUCCESS;
    }